addon/helpers/page-title.js

```diff
@@ -52,16 +71,17 @@ export default Helper.extend({
     let router = getOwner(this).lookup('router:main');
     let routes = router._routerMicrolib || router.router;
     let { activeTransition } = routes || {};
-    let headData = get(this, 'headData');
+    let document = get(this, 'document');
+    let isFastBoot = get(this, 'isFastBoot');
     if (activeTransition) {
       activeTransition.promise.finally(function () {
-        if (headData.isDestroyed) {
+        if (document.isDestroyed) {
```

Comment:
I don't think this will ever be set? We probably have to do `this.isDestroying` here

Reply:
Actually be the time this callback runs 'this' is already undefined. With tests we have at the moment. That's why reference are stored before finally callback definition. It's the destroy() method here not willDestroy().

Reply:
This will never be ￼defined here, need to change to arrow function.

Reply:
Oh my, I completely missed that. Yeah, then it will work just fine.